time time
mppi_ros/Config config

int64 step_count
float64 stage_cost
float64 reset_time
float64 optimization_time

mppi_ros/Array rollouts_cost
mppi_ros/Array weights
mppi_ros/Rollout[] rollouts
mppi_ros/Rollout optimal_rollout